config/default/manager_metrics_patch.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        args:
+        - "--metrics-addr=0.0.0.0:8080"
```

Comment:
@slopezz it means that ANY person can have access to your metrics it is not a good practice at all
You should ensure that your metrics endpoint is well protected.